packages/augur-sdk/src/state/Server.ts

```diff
@@ -1,13 +1,23 @@
+import { NetworkId } from "@augurproject/artifacts";
+import { EventEmitter } from "events";
+import { SDKConfiguration, startServer } from "./create-api";
+import { EndpointSettings } from "./getter/types";
 import * as HTTPEndpoint from "./HTTPEndpoint";
-import * as Sync from "./Sync";
 import * as WebsocketEndpoint from "./WebsocketEndpoint";
-import { EndpointSettings } from "./getter/types";
-import { EventEmitter } from "events";
 
 export async function run() {
   const settings = require("@augurproject/sdk/src/state/settings.json");
 
-  const api = await Sync.start(settings.ethNodeURLs[4], settings.testAccounts[0], false);
+  const config: SDKConfiguration = {
+    networkId: NetworkId.Kovan,
+    ethereum: {
+      http: settings.ethNodeURLs[4]
+    },
+    syncing: {
```

Comment:
The type marks this as optional. Can it be removed?

Reply:
`syncing` can be removed and it will use defaults.

Probably better to eschew defaults...